util/gh-pages/index.html

```diff
@@ -395,6 +495,18 @@ <h4 class="list-group-item-heading">
             // Get data
             $scope.open = {};
             $scope.loading = true;
+            // This will be used to jump into the source code of the version that this
+            // documentation is for. The version will be queried from the path as this
+            // works for all doc versions (master, beta, rust-1.52.0) with the exception
+            // of "stable" asa this is neither a tag or branch in our repo.
+            //
+            // The dirty hack is to link against beta in that case as that is the closes
+            // known branch to the current stable version. (The version would need to be
+            // queried) 
+            $scope.docVersion = window.location.pathname.split('/')[2];
+            if ($scope.docVersion === "stable") {
+                $scope.docVersion = "beta";
+            }
```

Comment:
The current stable release is saved in the ``stable`` symlink to the current stable. Don't know, if you can use that here.

Reply:
I wanted to discuss where I might get the current version from. Getting it from the symlink seems a bit complicated. The easiest solution would be to have a stable branch or maybe set a value during deployment that can easily be accessed in the script. 

The value is later used in this link template which works for tags and branches:
```
https://github.com/rust-lang/rust-clippy/tree/{$scope.docVersion}/clippy_lints
```
Examples:

| Lint list URL                                        | Corresponding code URL                                                 |
| ---------------------------------------------------- | ---------------------------------------------------------------------- |
| https://rust-lang.github.io/rust-clippy/master/      | https://github.com/rust-lang/rust-clippy/tree/master/clippy_lints      |
| https://rust-lang.github.io/rust-clippy/beta/        | https://github.com/rust-lang/rust-clippy/tree/beta/clippy_lints        |
| https://rust-lang.github.io/rust-clippy/rust-1.52.0/ | https://github.com/rust-lang/rust-clippy/tree/rust-1.52.0/clippy_lints |

Do you have a simple idea for this?

Reply:
We could add a `stable` branch. This branch would be just a dummy branch holding the commit of the latest release. Since we almost never do stable backports maintaining this shouldn't be much of an overhead.

Reply:
Leaving this as "unresolved", as a reminder for me to create a stable branch (I have to check that it doesn't break deployment and whatnot).

Reply:
Stable branch created: https://github.com/rust-lang/rust-clippy/tree/stable marking as resolved.